lib/gateway/errors.js

```diff
@@ -0,0 +1,28 @@
+const {GatewayError} = require('../errors');
+
+function errorHelper (req, res, next) {
+  res.egError = (code = 500, errors = []) => {
+    next(new GatewayError(code, errors));
+  };
+
+  next();
+}
+
+function notFoundHandler (req, res, next) {
+  res.egError(404);
```

Comment:
This seems a code smell to me - I have a `next` parameter that's not being called - but in the reality is being called by the inner method 🤔

Reply:
Fixed, I removed next